### PR TITLE
First HTML CSS Files

### PR DESCRIPTION
**Pull request by obibaadoma**
Installed html css linters and readme files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,11 @@
 # MicHelloProject
```
Comment (Ismailco) [inline README.md:1]: - [ ] Pull Request should have a summary describing introduced changes in more detail, kindly consider adding more detailed description about the changes in this PR

File: README.md [hunk 2]
```diff
@@ -1,20 +1,11 @@
...
-Hello Microverse
-Simple HTML CSS  Portfolio project
+First Hello Microverse Project
```
Comment (Ismailco) [inline README.md:3]: - [ ] [OPTIONAL] Kindly consider adding more details about the project to your README, you can add the `Getting Started` header with instructions on how to clone and access the project/repo

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @obibaadoma, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by V-Blaze — COMMENTED** [review]
Hi @obibaadoma  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, that has a merged pull request. A merged pull request will limit some GitHub features needed to perform a code review efficiently, I can see it seems you are repeating the module, I'll suggest you create a new pull request and submit it for a review. :smile: :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**